map-utils.js

```diff
@@ -1,126 +1,154 @@
-EARTH_RADIUS = 6371;
-TILE_SIZE = 256;
+var EARTH_RADIUS = 6371;
+var TILE_SIZE = 256;
 var tilePixelCenter = TILE_SIZE / 2;
-var pixelsPerLngDegree = TILE_SIZE / 360; 
+var pixelsPerLngDegree = TILE_SIZE / 360;
 var pixelsPerLngRadian = TILE_SIZE / (2 * Math.PI);
-var degreesToRadians = function (degrees) { 
-return (degrees * Math.PI) / 180 
-} 
-var radiansToDegrees = function (radians) { 
-return radians / (Math.PI / 180);
-} 
-var getDistance = function (posA, posB) { 
-var startLat = degreesToRadians(posA.lat || posA.latitude); 
-var startLng = degreesToRadians(posA.lng || posA.longitude);
-var endLat = degreesToRadians(posB.lat || posB.latitude);
-var endLng = degreesToRadians(posB.lng || posB.longitude);
-partialResult = Math.sin(startLat) * Math.sin(endLat) + Math.cos(startLat) * Math.cos(endLat) * Math.cos(startLng - endLng)
-console.log('partialResult: ', partialResult);
-distance = Math.acos(Math.min(partialResult, 1)) * EARTH_RADIUS 
-return Number(distance.toFixed(3)) 
-} 
-var getRadiusLatLng = function (latLng, radius) { 
-var startLat = degreesToRadians(latLng.lat) 
-var startLng = degreesToRadians(latLng.lng) 
-var dist = parseFloat(radius) / (EARTH_RADIUS * 1000) 
-var destAng = degreesToRadians(0) 
-var destLat = Math.asin(Math.sin(startLat) * Math.cos(dist) + Math.cos(startLat) * Math.sin(dist) * Math.cos(destAng)) 
-var destLng = ((startLng + Math.atan2(Math.sin(destAng) * Math.sin(dist) * Math.cos(startLat), Math.cos(dist) - Math.sin(startLat) * Math.sin(destLat))) * 180) / Math.PI 
-destLat = (destLat * 180) / Math.PI 
-console.log('destLat: ', destLat);
-return { lat: destLat, lng: destLng } 
-} 
-var getPointFromLatLng = function (latLng, opts) { 
-opts = opts || {asPlainObject: false};
-var siny = Math.max(Math.sin(degreesToRadians(latLng.lat)), -0.9999) 
-x = TILE_SIZE * (0.5 + latLng.lng / 360) 
-y = TILE_SIZE * (0.5 - Math.log((1 + siny) / (1 - siny)) / (4 * Math.PI)) 
-if (opts.asPlainObject) { 
-return {x: x, y: y} 
-}
-return new google.maps.Point(x, y)
-}
+var google;
```

Comment:
This works, although preferable would be to add a "globals" block to your configuration file that includes these. Same effect here.